psi4/driver/p4util/util.py

```diff
@@ -25,15 +25,18 @@
 #
 # @END LICENSE
 #
-
 """Module with utility functions for use in input files."""
 from __future__ import division
+#from __future__ import absolute_import
 import re
 import sys
 import os
 import math
 import numpy as np
+from datetime import datetime
+from uuid import uuid4
 from .exceptions import *
+from psi4.driver.procrouting.proc_util import check_iwl_file_from_scf_type
```

Comment:
Without reproducing the problem and experimenting, I can't venture the exact problem you're seeing, @edeprince3, but this line is my suspect for the problem, either because it's starting too high (`psi4.` not `.`) or because there's a circular import (symptom is some files work when others don't) or because this import should be hidden behind `def fcidump` (so doesn't get called upon `import psi4`).